src/components/News/index.js

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getNews } from '../../store/actions';
+import styles from './News.module.scss';
+
+export const News = () => {
+  const dispatch = useDispatch();
+  const news = useSelector(state => state.news);
+
+  useEffect(() => {
+    dispatch(getNews(5));
+  }, []);
+
+  return (
+    <div className={styles.newsContainer}>
+      {news?.map(item => 
+        <div className={styles.news}>
+          <div className={styles.title}>{item.title}</div>
+          <div className={styles.text}>{item.text}</div>
+        </div>
+      )}
```

Comment:
Missing `key` prop.

Missing wraps for function arguments.

Missing wraps for JSX.

This is how it should look like:

```javascript
      {news?.map((item) => (
        <div
          className={styles.news}
          key={item.id}
        >
          <div className={styles.title}>{item.title}</div>
          <div className={styles.text}>{item.text}</div>
        </div>
      ))}
```